src/backend/src/form-responses/form-responses.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { FormResponsesService } from './form-responses.service';
+import { CreateFormResponseDto, CreateMultiFormResponsesDto } from './dto';
+
+@Controller('form-responses')
+export class FormResponsesController {
+  constructor(private readonly formResponsesService: FormResponsesService) {}
+
+  @Post()
+  async create(
+    @Body('userId') userId: number,
+    @Body('questionId') questionId: number,
+    @Body('formResponse') formResponseData: CreateFormResponseDto,
```

Comment:
Ok here I don't think that this should take in a CreateFormResponseDto, I feel like it just needs to take in a string which is the response. Since it's answering a single question